test/hexpm_web/views/api/release_view_test.exs

```diff
@@ -0,0 +1,37 @@
+defmodule HexpmWeb.API.ReleaseViewTest do
+  use HexpmWeb.ConnCase, async: true
+
+  alias HexpmWeb.API.ReleaseView
+
+  import Hexpm.Factory
+
+  describe "show.json" do
+    test "includes publisher info" do
+      publisher =
+        build(:user,
+          username: "Publisher",
+          emails: [build(:email, email: "publisher@example.com", public: true)]
+        )
+
+      release =
+        build(:release,
+          package: build(:package),
+          requirements: [],
+          publisher: publisher
+        )
+
+      assert %{
+               "publisher" => %{
+                 "username" => "Publisher",
+                 "email" => "publisher@example.com"
+               }
```

Comment:
We usually test JSON responses in the controller tests, please move the assertion [here](https://github.com/hexpm/hexpm/blob/3e32d69f3b83fb19a9c21afaffb1b8336ac4b63e/test/hexpm_web/controllers/api/release_controller_test.exs#L35:L37).

Reply:
I've moved the test to there and removed the view test file. But I didn't assert the email/url rendering since they needed extra setup steps. Let me know if these two assertions are needed.